src/node_metadata.cc

```diff
@@ -21,6 +21,10 @@
 #include <nghttp3/version.h>
 #endif
 
+#if defined(__GLIBC__) && !defined(__UCLIBC__)
+#include <gnu/libc-version.h>  /* gnu_get_libc_version() */
```

Comment:
We don’t actually use this header here since we get the definition through `dlsym()`

Reply:
Great! I removed it in b62a6c0